### PR TITLE
add the location to each event in the schedule

### LINKED ISSUE #11: Add location to the schedule
The schedule should be updated to include the location of where things are happening.

### PR DESCRIPTION
**Pull request by bpas247**
Fixes #11

### CODE REVIEW

**Comment by Chris-Johnston** [comment]
Can you take a screenshot of what this looks like?
I had an idea in mind: we could make the third (location) column without a header, and make the location text smaller than the other text.

File: index.html [hunk 1]
```diff
@@ -129,46 +129,57 @@ <h2>
                     <tr>
                         <th>Time</th>
                         <th>Schedule</th>
+                        <th>Location</th>
                     </tr>
                     <tr>
                         <td>9:15 AM</td>
                         <td>Registration and Morning Coffee</td>
+                        <td>Activities & Recreation Center (ARC)</td>
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                        <td>North Creek Events Center</td>
```
this will be in the NCEC

File: index.html [hunk 2]
```diff
@@ -129,46 +129,57 @@ <h2>
...
                     </tr>
                     <tr>
                         <td>10:00 AM</td>
                         <td>Opening Ceremony</td>
+                        <td>Activities & Recreation Center (ARC)</td>
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                        <td>North Creek Events Center</td>
```

File: index.html [hunk 3]
```diff
@@ -129,46 +129,57 @@ <h2>
...
                     </tr>
                     <tr>
                         <td>11:15 AM</td>
                         <td>Intro to HTML, CSS, and JavaScript Workshop</td>
+                        <td>North Creek Events Center</td>
                     </tr>
                     <tr>
                         <td>12:00 Noon</td>
                         <td>Free Lunch</td>
+                        <td>Activities & Recreation Center (ARC)</td>
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                        <td>North Creek Events Center</td>
```

File: index.html [hunk 4]
```diff
@@ -129,46 +129,57 @@ <h2>
...
                     </tr>
                     <tr>
                         <td>2:45 PM</td>
                         <td>Cybersecurity Workshop</td>
+                        <td>North Creek Events Center</td>
                     </tr>
                     <tr>
                         <td>6:00 PM</td>
                         <td>Free Dinner</td>
+                        <td>Activities & Recreation Center (ARC)</td>
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                        <td>North Creek Events Center</td>
```

File: index.html [hunk 5]
```diff
@@ -129,46 +129,57 @@ <h2>
...
                     </tr>
                     <tr>
                         <td>9:15 PM</td>
                         <td>Project Presentations</td>
+                        <td>Activities & Recreation Center (ARC)</td>
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                        <td>North Creek Events Center</td>
```

File: index.html [hunk 6]
```diff
@@ -129,46 +129,57 @@ <h2>
...
                     </tr>
                     <tr>
                         <td>10:00 PM</td>
                         <td>Awards and Wrap-Up Presentation</td>
+                        <td>Activities & Recreation Center (ARC)</td>
```
Comment (Chris-Johnston) [inline index.html]:
```suggestion
                        <td>North Creek Events Center</td>
```

**Comment by bpas247** [comment]
something similar to this?
image

File: index.html [hunk 7]
```diff
@@ -129,46 +129,57 @@ <h2>
...
                         <th>Schedule</th>
+                        <th>Location</th>
                     </tr>
                     <tr>
                         <td>9:15 AM</td>
                         <td>Registration and Morning Coffee</td>
+                        <td>Activities & Recreation Center (ARC)</td>
                     </tr>
                     <tr>
                         <td>10:00 AM</td>
```
Comment (Chris-Johnston) [inline index.html]: we could add an entry for "Start Hacking!" that starts at 10:20 in the ARC. other than that, I think everything is in the NCEC

**Pull request closed without merging**